### PR TITLE
Added basic content to the Friends Page

### PR DESCRIPTION
**Pull request by salazw1**
## Issue Number
#150 

## Description

- Added static components to the Friends page that showcases current friends and recommended friends. 
- Created 2 example .json files to hold information of current and recommended friends

There is room for improvement, such as adding the profileIcon component that is currently being developed.

## Testing
Test cases need to be added for the added files; coverage for these files will be added later.

## Possible Errors
No possible errors known. 

image

### CODE REVIEW

**Review by nguyenj21 — APPROVED** [review]
Looks great! Good job. Maybe in the future if we have time, we can add more pretty details such as icons, avatars, and etc. so it becomes more colorful.

**Review by junxjr — APPROVED** [review]
I like the format for displaying the friend's page. It is very easy to read. Great job on keeping the same format as other pages using the data folder! The only thing I can suggest is to include some comments.

**Comment by salazw1** [comment]
> I like the format for displaying the friend's page. It is very easy to read. Great job on keeping the same format as other pages using the data folder! The only thing I can suggest is to include some comments.

oh I will definitely add more documentation for better readability!

File: small-talk/app/components/friends/individualFriend.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function IndividualFriend({ details }) {
```
Comment (aiharc1) [inline small-talk/app/components/friends/individualFriend.js:3]: I like the use of templates to make everything uniform!

File: small-talk/app/data/friendData/exampleFriendData.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+[
```
Comment (aiharc1) [inline small-talk/app/data/friendData/exampleFriendData.json:1]: I also like the separation of data files here

**Review by aiharc1 — APPROVED** [review]
Nice work here Warren. Checked out the page and everything looks sweeeeeet.

File: small-talk/app/data/friendData/exampleFriendData.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    {
```
Comment (39otsu) [inline small-talk/app/data/friendData/exampleFriendData.json:2]: If you wish to expand upon this, you can import this json directly into a mongodb collection and make an api to begin making requests

**Review by 39otsu — APPROVED** [review]
Looks really solid for a start. Lets get into talks about fetching the users from the database!

**Comment by AlvaroH2001** [comment]
Everything looks great, going to go ahead and merge this one.

**Pull request merged**